aiida/orm/utils/builders/code.py

```diff
@@ -222,3 +236,4 @@ def is_local(self):
     class CodeType(enum.Enum):
         STORE_AND_UPLOAD = 'store in the db and upload'
         ON_COMPUTER = 'on computer'
+        ON_CONTAINER = 'on container'
```

Comment:
```suggestion
        ON_CONTAINER = 'on container'
```
Would go with `CONTAINERIZED` or `CONTAINER` here.